File: CodeAssessment.FrontEnd/src/app/models/car-dto.ts
export class CarDto {
    public id: number;
    public make: string;
    public model: string;
    public year: number;
    public doors: number;
    public color: string;
    public price: number;
    public guess: number;
  
    constructor(id: number, make: string, model: string, year: number, doors: number, color: string, price: number, guess:number) {
      this.id = id;
      this.make = make;
      this.model = model;
      this.year = year;
      this.doors = doors;
      this.color = color;
      this.price = price;
      this.guess = guess;
    }
  }
  

File: CodingAssessment.Backend/CodingAssessment.Application/Features/Cars/CommonDto/CarDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CodingAssessment.Domain.Features.Cars;

namespace CodingAssessment.Application.Features.Cars.CommonDto
{
    public class CarDto
    {
        public int Id { get; set; }
        public string Make { get; set; }
        public string Model { get; set; }
        public int Year { get; set; }
        public int Doors { get; set; }
        public string Color { get; set; }
        public double Price { get; set; }

        public CarDto(Car car)
        {
            Id = car.Id;
            Make = car.Make;
            Model = car.Model;
            Year = car.Year;
            Doors = car.Doors;
            Color = car.Color;
            Price = car.Price;
        }
    }
}
